CommonInstall/util_init.pl

```diff
@@ -346,6 +351,7 @@ ()
 			"rocky" => "redhat",
 			"almalinux" => "redhat",
 			"circle" => "redhat",
+			"ocs" => "opencloudos",
```

Comment:
for redhat-like OS we should use redhat for the vendor, just like centos and rocky do.

See https://github.com/intel/eth-fast-fabric/pull/1/commits/97d8054919c8137a3d56167e011c1c05e5b4c9f3

Reply:
We have something in common, but we are not based on redhat and are not redhat-like exactly.